R/revdepcheck.R

```diff
@@ -134,7 +134,7 @@ revdep_install <- function(pkg = ".", quiet = FALSE) {
     with_libpaths(
       dir_find(pkg, "old"),
       with_options(
-        list(warn = 2),
+        warn = 2,
```

Comment:
This is not good, I think, see `withr::with_options`.

Reply:
The rlang version take spliceable dots.

Reply:
Oh, so rlang has  `with_` functions, I did not know that.  I am pretty sure that I miss this at some point in the future, can please add an explicit `rlang::` here, and below?

Reply:
Done.